bigquery_etl/shredder/config.py

```diff
@@ -238,6 +238,12 @@ def fields(self) -> tuple[str, ...]:
     client_id_target(table="telemetry_stable.untrusted_modules_v4"): DESKTOP_SRC,
     client_id_target(table="telemetry_stable.update_v4"): DESKTOP_SRC,
     client_id_target(table="telemetry_stable.voice_v4"): DESKTOP_SRC,
+    client_id_target(table="telemetry_derived.mobile_engagement_clients_v1"): (
```

Comment:
@BenWu If I recall correctly uou made a change to automatically detect client tables that need to be shredded, is this correct?

Reply:
The `find_glean_targets` function in this file detects glean tables but that only works with the generated glean datasets like `org_mozilla_firefox_stable`, `fenix_derived`.  This needs to be added because it's in telemetry